lstchain/irf/hdu_table.py

```diff
@@ -309,13 +307,16 @@ def create_event_list(
 
     src_sky_pos = SkyCoord(
         alt=reco_alt, az=reco_az, frame=AltAz(obstime=time_utc, location=location)
-    ).transform_to(frame="icrs")
+    )
     tel_pnt_sky_pos = SkyCoord(
-        alt=pointing_alt.mean(),
-        az=pointing_az.mean(),
-        frame=AltAz(obstime=mean_time, location=location),
+        alt=pointing_alt[0],
+        az=pointing_az[0],
+        frame=AltAz(obstime=time_utc[0], location=location),
     ).transform_to(frame="icrs")
 
+    with erfa_astrom.set(ErfaAstromInterpolator(30 * u.s)):
```

Comment:
I typically use five minutes now. See https://docs.astropy.org/en/stable/coordinates/index.html#improving-performance-for-arrays-of-obstime

Reply:
Yes, I tried to check with varying the time period, and although the time taken to transform does not differ much, the min/max separation was lower.
Although, I do not think it would be too significant for our use case. 
Is 30 seconds too low?

Reply:
shouldn't really matter